django/forms/formsets.py

```diff
@@ -450,29 +464,32 @@ def media(self):
         else:
             return self.empty_form.media
 
+    def get_context(self):
+        return {'formset': self}
+
+    def render(self, template_name=None, context=None, renderer=None):
+        return mark_safe((renderer or self.renderer).render(
+            template_name or self.template_name,
+            context or self.get_context(),
+        ))
+
```

Comment:
As `BaseFormSet` is inheriting from `Renderable` we can ditch this as the definition is the same:
```suggestion
```
You can also remove `.as_table()`, `.as_p()`, and `.as_ul()`.

Reply:
🤦